compositor_render/src/registry.rs

```diff
@@ -11,6 +11,14 @@ pub enum RegisterError {
     },
 }
 
+#[derive(Debug, thiserror::Error)]
+#[error("Failed to unregister a {item_type}. The \"{renderer_id}\" {item_type} does not exist.")]
+pub struct UnregisterError {
+    item_type: &'static str,
+    renderer_id: RendererId,
+}
```

Comment:
```suggestion
pub struct UnregisterItemNotFoundError {
    item_type: &'static str,
    renderer_id: RendererId,
}
```

There are a lot of other unregister errors

Reply:
I would prefer to leave it like this for consistency with RegisterError.